vue.config.js

```diff
@@ -1,30 +1,9 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
-
-const webpack = require('webpack')
-const fs = require('fs')
-const packageJson = fs.readFileSync('./package.json')
-const appName = JSON.parse(packageJson).appName
-const appVersion = JSON.parse(packageJson).version
-const sbcName = JSON.parse(packageJson).sbcName
-const sbcVersion = JSON.parse(packageJson).dependencies['sbc-common-components']
-const aboutText1 = (appName && appVersion) ? `${appName} v${appVersion}` : ''
-const aboutText2 = (sbcName && sbcVersion) ? `${sbcName} v${sbcVersion}` : ''
-
 module.exports = {
```

Comment:
Remove vue.config.js?

Reply:
Project runs but getting a fatal error when I run unit tests:
![fatal error unit tests](https://github.com/bcgov/business-edit-ui/assets/122301442/85b56842-baf2-41c4-96c6-145aa478b5d1)

Reply:
This file looks the same as in Create UI so I'm OK with this.